pelita/game.py

```diff
@@ -788,6 +788,56 @@ def play_turn(game_state, allow_exceptions=False):
 
     return game_state
 
+def bot_in_homezone(team, bot_position, boundary):
+    if team == 0:
+        return bot_position[0] < boundary
+    elif team == 1:
+        return bot_position[0] >= boundary
+
+def check_kill_death(team, turn, boundary, game_state):
```

Comment:
Same here; not really a fan of exposing the somewhat ill-defined boundary (is it on the last blue column or the first red column?). If we pass the game_state, we already have all we need.